CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## Latest version of the provider was tested on Artifactory 7.32.1
```

Comment:
This is good. However, I was thinking we need to attach the Artifactory version next to the release version/date. This way, users will know which version of the provider works with which version of Artifactory.

Reply:
Changed, thanks for the help!